README.md

```diff
@@ -58,6 +56,12 @@ If you're creating a new topic, keep the following in mind as well:
 | reference | Any reference-type article that isn't auto-generated. |
 | article | Use for community-contributed content (that is, anything from outside the engineering team or the content team at Microsoft. |
 
+## Merging to the live branch
+
+Periodically, about once a month, a manual RI is performance from `main` -> `live` branch.
```

Comment:
```suggestion
NuGet team members have permissions to merge to `live` branch manually, at their own discretion.
Otherwise, the regular (about once a month) Reverse Integration (RI) will be performed from `main` -> `live` branch.
```